pkg/systemd/generate/common.go

```diff
@@ -8,11 +8,6 @@ import (
 	"github.com/pkg/errors"
 )
 
-// minTimeoutStopSec is the minimal stop timeout for generated systemd units.
-// Once exceeded, processes of the services are killed and the cgroup(s) are
-// cleaned up.
-const minTimeoutStopSec = 60
```

Comment:
Is this safe? If we generated a unit without --new we still use `podman stop`. Given the the unit timeout now matches the podman stop timeout I think it is possible that systemd kills the podman stop process.

Reply:
OK I just tried, this it is **not safe**! Systemd killed the podman stop process and now the container is stuck in stopping state.

Reply:
Very fair point. Thanks for catching!